api/clients/codecs/blob_codec.go

```diff
@@ -4,22 +4,22 @@ import (
 	"fmt"
 )
 
-type BlobEncodingVersion byte
+type PayloadEncodingVersion byte
 
 const (
-	// This minimal blob encoding contains a 32 byte header = [0x00, version byte, uint32 len of data, 0x00, 0x00,...]
+	// PayloadEncodingVersion0 entails a 32 byte header = [0x00, version byte, big-endian uint32 len of payload, 0x00, 0x00,...]
```

Comment:
can you add comment as to why we add the first 0x00 byte (so that it gets parsed as a bn254 FE). Took me a second to remember this

Reply:
[78010448](https://github.com/Layr-Labs/eigenda/pull/1120/commits/7801044891e39e9c9ed73b1404af20164c26a228)